### ISSUE TITLE
Can‘t reproduce the results for ’MobileNetv2Plus ‘ 

### ISSUE THREAD

**Issue by idealwei** [irfanICMLL/structure_knowledge_distillation#13]
Hi，
I  kept the default setting of this code and replace the student model with MobileNetV2Plus in LightNet. However, the results with distillation is even worse than that without. Can you show me more details about how to train with MobileNetV2Plus ? It would be very helpful.

**Comment by irfanICMLL** [comment]
You can use the original training settings in LightNet. Or you can try to adjust the loss weight in your situation. It is wired that the distillation results are worse. We always get a better results with the distillation items.
